src/components/CountriesList.js

```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import { Country } from './Country';
+import StatusType from './StatusTypes';
+
+function CountriesList() {
+  const countries = useSelector((store) => store.countries);
+
+  if (!countries || countries.status === StatusType.LOADING) {
+    return <span className="span2">Loading...</span>;
+  }
+
+  return (
+    <section>
+      <ul>
+        {countries?.countriesArr
+          .filter(({ countryName }) => countryName
+            .toUpperCase()
+            .includes(countries.searchString.toUpperCase()))
+          .map((country, index) => (
+            <Country
+              key={country.geonameId}
+              index={index}
+              geonameId={country.geonameId}
+              countryName={country.countryName}
+              countryCode={country.countryCode}
+              currencyCode={country.currencyCode}
+              areaInSqKm={country.areaInSqKm}
+              continentName={country.continentName}
+              population={country.population}
+            />
+          ))}
+      </ul>
+    </section>
```

Comment:
- [ ] You have done an impressive job with this implementation, @melkamu12 :clap: 
- The app looks really cool and better but the only issue we currently have is that it not looking so great on the mobile screen. Somehow, the country cards are not equal when the screen gets smaller and there seems so be a behavior that is proving unpredictable: The styles on the header part seems not to be consistent. Kindly check to ensure that it maintains an 100% width in all devices.
- I left a very detailed video for you [here](https://watch.screencastify.com/v/vbo20972RNIRdkZ9voiV), kindly check it out and apply the fix.
_You are almost done with this, and you are doing so well already!_

Reply:
I corrected the responsibility and the card height is equal ` min-width: 105px;
  max-width: 100%;
  background-color: var(--color-c9bcd719);
  display: flex;
  height: 150px;`